src/html.rs

```diff
@@ -0,0 +1,137 @@
+use maud::{html, Markup, DOCTYPE};
+use qr_code::QrCode;
+use std::io::Cursor;
+
+const CSS: &str = include_str!("../assets/index.css");
+
+fn inner_header() -> Markup {
+    let header = html! {
+        nav.navbar.navbar-expand-lg.fixed-top.navbar-dark.bg-dark {
+            a.navbar-brand href="#" { "BTC Tip Server" }
+            a.navbar-brand href="https://github.com/lvaccaro/btctipserver" {
+                i.bi.bi-github role="img" aria-label="GitHub" {}
+            }
+        }
+    };
+    header
+}
+
+fn inner_network(network: &str) -> Markup {
+    let partial = html! {
+        div class="d-flex align-items-center p-3 my-3 text-white-50 bg-orange rounded box-shadow" {
```

Comment:
Would be nice to use a different background color for non-main networks, some use blue for testnet/regtest.

Reply:
thanks for the suggestion... add color class parameter based on network type